src/AudacityApp.cpp

```diff
@@ -384,6 +384,11 @@ void PopulatePreferences()
       gPrefs->Write(wxT("/GUI/Toolbars/Time/Show"),1);
    }
 
+   if (std::pair{ vMajor, vMinor } < std::pair{ 3, 1 } ) {
```

Comment:
I am understood what you do, but I am not understand why we should to do this?

Reply:
Run Audacity 3.0.4 or earlier and exit.  It makes audacity.cfg which remembers preferences and other settings.

Then run my branch, but without the line above.

You will see that the control toolbar is too narrow to show the new button completely.

This line fixes that.